src/credentials/FabricTable.cpp

```diff
@@ -2080,4 +2080,19 @@ CHIP_ERROR FabricTable::GetFabricLabel(FabricIndex fabricIndex, CharSpan & outFa
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricTable::GetNextFabricAdditionIndex(FabricIndex * nextIndex)
+{
+    EnsureNextAvailableFabricIndexUpdated();
+    if (!mNextAvailableFabricIndex.HasValue())
+    {
+        return CHIP_ERROR_NO_MEMORY;
```

Comment:
IS CHIP_ERROR_NO_MEMORY the right return error here? What about CHIP_ERROR_INVALID_FABRIC_INDEX?

Reply:
No memory is likely the correct one.

Reply:
Leaving this as CHIP_ERROR_NO_MEMORY.